common/scala/src/main/scala/whisk/core/entity/Exec.scala

```diff
@@ -43,40 +47,50 @@ import whisk.core.entity.ArgNormalizer.trim
  *         jar   : a base64-encoded JAR file when kind is "java",
  *         name  : a fully-qualified class name when kind is "java" }
  */
-sealed abstract class Exec(val kind: String) {
+sealed abstract class Exec(val kind: String) extends ByteSizeable {
     def image: String
+    def size: ByteSize
```

Comment:
since this is defined in the trait, you don't need to red an abstract method here.